pkg/virtual/workspaces/cache/cache.go

```diff
@@ -27,49 +27,49 @@ import (
 	klog "k8s.io/klog/v2"
 
 	workspaceapi "github.com/kcp-dev/kcp/pkg/apis/tenancy/v1alpha1"
-	workspaceClient "github.com/kcp-dev/kcp/pkg/client/clientset/versioned/typed/tenancy/v1alpha1"
+	kcpclient "github.com/kcp-dev/kcp/pkg/client/clientset/versioned"
 )
 
 // NewClusterWorkspaceCache returns a non-initialized ClusterWorkspaceCache. The cache needs to be run to begin functioning
-func NewClusterWorkspaceCache(workspaces cache.SharedIndexInformer, client workspaceClient.ClusterWorkspaceInterface, defaultNodeSelector string) *ClusterWorkspaceCache {
+func NewClusterWorkspaceCache(workspaces cache.SharedIndexInformer, kcpClusterInterface kcpclient.ClusterInterface, defaultNodeSelector string) *ClusterWorkspaceCache {
 	if err := workspaces.GetIndexer().AddIndexers(cache.Indexers{
 		"requester": indexWorkspaceByRequester,
 	}); err != nil {
 		panic(err)
 	}
 	return &ClusterWorkspaceCache{
-		Client:              client,
+		ClusterInterface:    kcpClusterInterface,
```

Comment:
kcpClusterClient, also above. Not *Interface

Reply:
Fixed in commit d85f6e867f5bc15b9948d1193a9b635646a7a09a